documentation/docs/steps/abapEnvironmentPullGitRepo.md

```diff
@@ -24,14 +24,36 @@ abapEnvironmentPullGitRepo script: this
 If you want to provide the host and credentials of the Communication Arrangement directly, the configuration could look as follows:
 
 ```yaml
-
 steps:
   abapEnvironmentPullGitRepo:
     repositoryNames: ['/DMO/GIT_REPOSITORY']
     abapCredentialsId: 'abapCredentialsId'
     host: '1234-abcd-5678-efgh-ijk.abap.eu10.hana.ondemand.com'
 ```
 
+Also you can specify a dedicated file, e.g. `repositories.yml` containing the repositories to be pulled:
+
+```yaml
+steps:
+  abapEnvironmentPullGitRepo:
+    repositories: 'repositories.yml'
+    abapCredentialsId: 'abapCredentialsId'
+    host: '1234-abcd-5678-efgh-ijk.abap.eu10.hana.ondemand.com'
+```
+
+The associated config file, e.g. `repositories.yml` could look as follows:
+
+```yaml
+repositories:
+- name: '/DMO/GIT_REPOSITORY'
+  branch: 'master'
```

Comment:
I would leave out "branch". The branch has no meaning for this step (and we should not suggest that it does).